linux/check_efi_firmware/check_efi_firmware.yml

```diff
@@ -31,7 +31,7 @@
         - include_tasks: ../../common/vm_wait_log_msg.yml
           vars:
             vm_wait_log_name: "vmware.log"
-            vm_wait_log_msg: "The EFI ROM is {{ bitness }}"
+            vm_wait_log_msg: "The EFI ROM is .* {{ bitness }}"
```

Comment:
It should have no space following .*, otherwise it won't match The EFI ROM is 64-bit.

Reply:
By the way, can we add a task to fetch vmware.log in rescue block if check_efi_firmware failed?

Reply:
Test this it works.
```
TASK [debug] *****************************************************************************************************************************
ok: [localhost] => {
    "msg": [
        "Found 'The EFI ROM .* 64-bit' message in VM log vmware.log: Success",
        "Found logs list: ['The EFI ROM is 64-bit']"
    ]
}
```

Reply:
> By the way, can we add a task to fetch vmware.log in rescue block if check_efi_firmware failed?

This is another task or issue we can address for all the cases or some required cases.